terraform_files/none/consolemodel.xsl

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" ?>
```

Comment:
What does it do? Please elaborate on what's the benefit from adding this feature as part of the PR description

Reply:
there is nothing new , it is just copy paste from BM platform functionality